src/domains/ecommerce-app/04_apim_ecommerce_io.tf

```diff
@@ -0,0 +1,121 @@
+##############
+## Products ##
+##############
+
+module "apim_ecommerce_io_product" {
```

Comment:
why new product?

Reply:
For consistency with Checkout which has its own [dedicated product](https://github.com/pagopa/pagopa-infra/blob/main/src/domains/ecommerce-app/04_apim_ecommerce_checkout.tf#L5). Maybe we could rename this into `auth` instead of referring explicitly to IO?

Reply:
mmm I don't like the dedicated checkout/IO product with eCommerce, but I didn't remember 😁. We have one product for eCommerce, one for eCommerce checkout and one for the IO app. The advantage is not clear to me, but we continue with that approach.
we could refine and consider collapsing everything under the same product, eCommerce in a dedicated PR.